.build/tasks/New-Release.GitHub.build.ps1

```diff
@@ -65,16 +64,19 @@ task Publish_release_to_GitHub -if ($GitHubToken -and (Get-Module -Name PowerShe
 
     "`tProject Path                  = $ProjectPath"
 
-    # find Module's nupkg
-    $PackageToRelease = Get-ChildItem (Join-Path $OutputDirectory "$ProjectName.$moduleVersion.nupkg")
+    # find Module's nupkg if it exists
+    $packagedProjectNupkg = Join-Path -Path $OutputDirectory -ChildPath "$ProjectName.$moduleVersion.nupkg"
+    $PackageToRelease = Get-ChildItem -Path $packagedProjectNupkg -ErrorAction Ignore
+    # If the Project nupkg is not found, don't fail. You can still create a release and specify the
+    # assets in the build.yml (i.e. Chocolatey packages or Azure Policy Guest Config Packages)
     $ReleaseTag = "v$ModuleVersion"
 
     Write-Build DarkGray "About to release '$PackageToRelease' with tag and release name '$ReleaseTag'"
     $remoteURL = git remote get-url origin
 
     if ($remoteURL -notMatch 'github')
```

Comment:
Not sure this is needed since the GitHubToken need to be set to even run the task? 🤔

Reply:
Good point.